src/daemon/globals.c

```diff
@@ -0,0 +1,13 @@
+#include "globals.h"
```

Comment:
This file needs a license header. In case of the core daemon, we insist on the [MIT license](https://opensource.org/licenses/MIT).

Dito for `src/daemon/globals.h`.